Frontend/src/components/General/Navbar.js

```diff
@@ -0,0 +1,46 @@
+import React from  "react"
+import "./Navbar.css"
+import logo from "./../../Resources/logo.png"
+
+export default function Navbar({ buttonSet }) {
```

Comment:
for `{ buttonSet }` change this to `props` -> this is the convention for react. Then, update all calls of `buttonSet` to `props.buttonSet`